src/mono/wasm/Makefile

```diff
@@ -112,6 +112,9 @@ clean:
 # Helper targets
 .PHONY: runtime
 
+timezone-data:
+	cp runtime/dotnet.timezones.blat $(BUILDS_BIN_DIR)
```

Comment:
nit in case you touch this again: this should ideally use the filename as the target name instead so make's dependency tracking works (right now it always copies the file even if it's already there:

```make
$(BUILDS_BIN_DIR)/dotnet.timezones.blat: runtime/dotnet.timezones.blat
	cp runtime/dotnet.timezones.blat $@
```

(the `$@` is a special variable that means the target filename: https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html)

Reply:
good to know. i can open another pr with this. thank you!